src/utils/sorter.js

```diff
@@ -0,0 +1,18 @@
+import dayjs from 'dayjs';
+
+function sortByDay(a, b) {
+
+  return new Date(a.dateFrom).getTime() - new Date(b.dateFrom).getTime();
+}
```

Comment:
Лишний отступ, другого кода нет - можно не отделять return
```suggestion
function sortByDay(a, b) {
  return new Date(a.dateFrom).getTime() - new Date(b.dateFrom).getTime();
}
```